src/overview/tabs/Health/cards/ConditionsCard.tsx

```diff
@@ -2,23 +2,27 @@ import type { FC } from 'react';
 import { ConditionsSection } from 'src/modules/Providers/views/details/components/ConditionsSection/ConditionsSection';
 import { useForkliftTranslation } from 'src/utils/i18n';
 
-import type { V1beta1ForkliftController } from '@kubev2v/types';
+import type { K8sResourceCondition, V1beta1ForkliftController } from '@kubev2v/types';
 import { Card, CardBody, CardTitle } from '@patternfly/react-core';
 
 type ConditionsCardProps = {
   obj?: V1beta1ForkliftController;
-  loaded?: boolean;
-  loadError?: unknown;
+};
+
+type ConditionsStatus = {
+  conditions: K8sResourceCondition[];
 };
```

Comment:
type ConditionsCardProps = {
  obj?: V1beta1ForkliftController & {
  conditions: K8sResourceCondition[];
}
}; 

then u can remove the as